packages/aurora-erc20/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Happy to see this, but looks like it is still not used.

Reply:
Because the code structure is similar with other connector packages which are also js. I plan to finish adding typescript to the connector packages once the `ethersjs` transition is complete.